src/components/forms/Checkbox/Checkbox.tsx

```diff
@@ -179,7 +179,7 @@ const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
 Checkbox.propTypes = {
   name: PropTypes.string.isRequired,
   checkboxId: PropTypes.string,
-  label: PropTypes.string,
+  label: PropTypes.node || PropTypes.string,
```

Comment:
no need to provide both. `PropTypes.node` also covers strings you can take a look into the TS type for that
```
export type ReactNodeLike =
    | {}
    | ReactElementLike
    | ReactNodeArray
    | string
    | number
    | boolean
    | null
    | undefined;

Reply:
You're totally right. If we keep both will be redundant.